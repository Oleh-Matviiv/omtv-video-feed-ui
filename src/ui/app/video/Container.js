import React from 'react';
import styled from 'styled-components';

const Container = styled.article.attrs({
  className: 'video',
})`
  border: .1rem solid #efefef;
  margin-bottom: 2rem;
  display: grid;
  grid-template-areas: "title views"
                       "play play";
`;

export default Container;
