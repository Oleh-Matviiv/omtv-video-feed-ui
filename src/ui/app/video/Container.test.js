import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import Container from './Container';

describe('snapshots', () => {
  it('Container', () => {
    const wrapper = mount(
      <Container>children</Container>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
