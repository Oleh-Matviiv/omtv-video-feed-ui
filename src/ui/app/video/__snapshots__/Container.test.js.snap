// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots Container 1`] = `
.c0 {
  border: .1rem solid #efefef;
  margin-bottom: 2rem;
  display: grid;
  grid-template-areas: "title views" "play play";
}

<styled.article>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [
          Object {
            "className": "video",
          },
        ],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  border: .1rem solid #efefef;
  margin-bottom: 2rem;
  display: grid;
  grid-template-areas: \\"title views\\"
                       \\"play play\\";
",
          ],
        },
        "displayName": "styled.article",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "article",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <article
      className="video c0"
    >
      children
    </article>
  </StyledComponent>
</styled.article>
`;
